2017年4月11日のアプリアップデート対応。アクセサリーの分類変更

koala31 opened the pull request.

koala31 committed:
ヘアアクセサリー -> ヘッドアクセ

koala31 committed:
ヘアアクセ・ベール -> ヴェール

koala31 committed:
ヘアアクセ・追加 -> カチューシャ

koala31 committed:
首飾り・ストール -> マフラー

koala31 committed:
首飾り -> ネックレス

koala31 committed:
腕飾り・右手 -> 右手飾り

koala31 committed:
腕飾り・左手 -> 左手飾り

koala31 committed:
腕飾り・手袋両手物 -> 手袋

koala31 committed:
六番目の契約書 を 両手持ち に

koala31 committed:
手持品・右手 -> 右手持ち

koala31 committed:
手持品・左手 -> 左手持ち

koala31 committed:
特殊・眼鏡 -> フェイス

koala31 committed:
特殊・肩掛け -> ボディ

koala31 committed:
特殊・タトゥ -> タトゥー

koala31 committed:
特殊・羽 -> 羽根

koala31 committed:
特殊・しっぽ -> しっぽ

koala31 committed:
特殊・落物吊物吐息 -> 前景

koala31 committed:
特殊・マスコット -> 後景

koala31 committed:
特殊・敷物 -> 床

koala31 committed:
特殊・運命の箱 -> 吊り

koala31 committed:
FormatVersion 4 から 5 に変更

整理番号89901から記述している種類の一覧をアプリの並びにしました。

koala31 commented:
関連 Issues #90  
関連 Wiki [アクセサリーの分類変更](https://github.com/koala31/MiracleNikkiJp_items/wiki/%E3%82%A2%E3%82%AF%E3%82%BB%E3%82%B5%E3%83%AA%E3%83%BC%E3%81%AE%E5%88%86%E9%A1%9E%E5%A4%89%E6%9B%B4)

ShTair commented:
首飾り→ネックレスの変更で、アイテム名まで変わっちゃってるのがいくつかあります。

koala31 committed:
名前欄の修正（名前の欄まで文字列変換の対象としていた）

koala31 commented:
ご指摘ありがとうございます。うっかりしてました。  
commit e6b7202ef2b32104f4262b00a26c56bf735760cb で、名前の欄は、旧版を上書きしたので  
今度は大丈夫と思います。

ShTair commented on MiracleNikkiJp_items.csv:
```diff
@@ -71508,15 +71508,15 @@ FormatVersion,4,靴下の種類を細分化しました,,,,,,,,,,,,,,,
+71505,1505,後景,←左にアクセサリーの種類を書きます,悪夢の診察室,5,A,,,SS,,SS,,S,A,,メディカル,
+71506,1506,前景,←左にアクセサリーの種類を書きます,7号標本,5,S,,S,,SS,,S,,A,,,白
+71507,1507,フェイス,←左にアクセサリーの種類を書きます,虚妄の診察室,5,S,,SS,,S,,S,,,S,メディカル,白
+71508,1508,両手持ち,←左にアクセサリーの種類を書きます,六番目の契約書<両手>,5,A,,S,,SS,,SS,,A,,メディカル,
```
「六番目の契約書」の名前の、<両手>は残しておきますか？

koala31 replied:
あ！これ不要ですね。消しておきます。

koala31 committed:
両手持ち一時措置の終了。六番目の契約書<両手> -> 六番目の契約書

「両手持ち」が独立した分類となりました。
両手持ちを着用する場合は、右手持ちも左手持ちも着用できません。
この特徴は本データに表現されていませんので利用側で配慮してください。

ShTair commented:
お疲れさまです。
またそのうち更新分プルリクエストします。
